src/test/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepositorySpek.kt

```diff
@@ -121,6 +118,51 @@ class AktivitetskravRepositorySpek : Spek({
                     } shouldBe true
                 }
 
+                it("Is not retrieving expired varsler which has OPPFYLT or UNNTAK status after they are created") {
```

Comment:
Jeg tror disse testene kan bli skrevet en del enklere når vi får alle persistens funksjoner inn i `Repository` filer. Slik at vi slipper å lage custom test funksjoner som sørger for at vi lager en vurdering etter at et aktivitetskrav er oppdatert med denne vurderingen osv.

Reply:
Vi burde vel egentlig hatt en `AktivitetskravVarselRepositorySpek` for noen av disse testene her.

Reply:
Ja enig, kan refaktorere i en annen PR 😊